doc/source/community/resources/2022_IS2_HW.rst

```diff
@@ -0,0 +1,17 @@
+.. _resource_IS2HW_2022:
+
+Third ICESat-2 Hackweek Facilitated by the University of Washington eScience Institute
+--------------------------------------------------------------------------------------
+This event is currently being planned, and will take place 21-25 March 2022.
+Please see the `event website <https://icesat-2hackweek.github.io/>`_ for more information.
```

Comment:
```suggestion
Please see the `event website <https://icesat-2.hackweek.io/>`_ for more information.
```

Reply:
Are there any plans for the former hackweek url? I've been going to that one by mistake over the past week.